cmd/skopeo/sync.go

```diff
@@ -280,21 +280,21 @@ func imagesToCopyFromRegistry(registryName string, cfg registrySyncConfig, sourc
 		}).Info("Processing repo")
 
 		var sourceReferences []types.ImageReference
-		for _, tag := range tags {
-			source := fmt.Sprintf("%s:%s", repoName, tag)
-
-			imageRef, err := docker.ParseReference(source)
-			if err != nil {
-				logrus.WithFields(logrus.Fields{
-					"tag": source,
-				}).Error("Error processing tag, skipping")
-				logrus.Errorf("Error getting image reference: %s", err)
-				continue
+		if len(tags) != 0 {
```

Comment:
If the tag name of the mirror is misspelled and does not conform to the specification, it will be skipped

Reply:
This is the “Clarify control flow when handling the tags list” commit:
> Don't run the the "tags specified" loop when tags should be discovered.

It’s just a cleanup for clarity, to match the control flow with the design (so that the `len(tags) == 0` path does not execute the “use these listed tags” code).